fe/model.py

```diff
@@ -10,10 +12,23 @@
 from ff import Forcefield
 from md.barostat.utils import get_bond_list, get_group_indices
 
-from parallel.client import AbstractClient
-from typing import Optional
+from parallel.client import AbstractClient, SerialClient
+from typing import Optional, List, Tuple
 
 
+def apply_hmr(masses, bond_list, multiplier=2):
+
+    def is_hydrogen(i):
+        return np.abs(masses[i] - 1.00794) < 1e-3
+
+    for i, j in bond_list:
+        i, j = np.array([i, j])[np.argsort([masses[i], masses[j]])]
+        if is_hydrogen(i) and not is_hydrogen(j):
+            # H-X
+            # order of operations is important!
+            masses[j] -= multiplier*masses[i]
+            masses[i] += multiplier*masses[i]
+    return masses
```

Comment:
Question: Should this utility live in `fe/` or in `md/`?

Documentation request: add reference for which HMR scheme is used.

Reply:
I put this in `fe/model.py` as I had seen that @proteneer had this implementation in `fe/model_utils.py`. I would think the `md` directory is the right place, and if you could comment @proteneer so that we can have one place for our two branches

Reply:
I'm okay with this going where-ever - I'll just import it. I think `md/` better as well.

Reply:
Use this link: https://pubmed.ncbi.nlm.nih.gov/26574392/

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/451/commits/5e594209bb81d9f9ed725df0af710e53fa8577d7